Intersect.Server/Core/CommandParsing/CommandParser.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
Revert changes to this file, looks like BOM

Reply:
Done